src/main/java/com/volasoftware/tinder/service/UserServiceImpl.java

```diff
@@ -167,11 +175,17 @@ public void updateUserPassword(ChangePasswordDto changePasswordDto) {
         userRepository.save(user);
     }
 
-    private User getLoggedUser() {
-        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
-        String currentUser = authentication.getName();
+    public void addFriend(Long userId, Long friendId) {
```

Comment:
Can we make this method to use single parameter only friendId and take the user from the security context, then persist the changes like adding friend and removing it?